corehq/form_processor/models.py

```diff
@@ -1070,6 +1074,7 @@ class CaseAttachmentSQL(PartitionedModel, models.Model, SaveStateMixin, IsImageM
 
     # DEPRECATED - use CaseAttachmentSQL.content_md5() instead
     md5 = models.CharField(max_length=255, default="")
+    type_code = models.IntegerField(default=CODES.form_attachment)
```

Comment:
I don't think this is necessary as `CaseAttachments` will always be `CODES.form_attachment`. Also we don't really support case attachments (it's on the kill list)